Include/opcode.h

```diff
@@ -149,6 +149,7 @@ extern "C" {
 #define LOAD_GLOBAL_ADAPTIVE     41
 #define LOAD_GLOBAL_MODULE       42
 #define LOAD_GLOBAL_BUILTIN      43
+#define NEED_OPCODE_JUMP_TABLES
 #ifdef NEED_OPCODE_JUMP_TABLES
 static uint32_t _PyOpcode_RelativeJump[8] = {
```

Comment:
Hummm, apparently there is a bunch of warnings regarding the fact that `_PyOpcode_RelativeJump` is not used now. Is this due to these changes or was there before?

Reply:
Maybe this PR can do that if you don't regen.